consensus/src/dag/bootstrap.rs

```diff
@@ -102,14 +123,19 @@ impl TDagMode for ActiveMode {
         });
 
         // Run the network handler until it returns with state sync status.
-        let sync_status = self.handler.run(dag_rpc_rx).await;
+        let sync_outcome = self.handler.run(dag_rpc_rx, self.buffer).await;
 
-        match sync_status {
-            StateSyncStatus::NeedsSync(certified_node_msg) => Some(Mode::Sync(SyncMode {
+        info!(
+            LogSchema::new(LogEvent::SyncOutcome),
+            sync_outcome = %sync_outcome,
```

Comment:
this seems redundant log of the one in SyncMode?

Reply:
This also covers the EndsEpoch case.